src/styles/_catalog.scss

```diff
@@ -1,22 +1,33 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: $card-width;
+  justify-content: center;
+  gap: 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
-    height: 300px;
+    display: grid;
+    place-items: center;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(1.2);
```

Comment:
You should specify transition properties directly
Default is all, but it could affect old browsers' performance